composer.json

```diff
@@ -54,14 +54,16 @@
         "complete-check": [
             "@check-cs",
             "phpunit",
-            "@phpstan"
+            "@phpstan",
+            "format-documentation"
```

Comment:
Maybe `update-docs` would be better, since it only generates 1 new file and doesn't format documentation at all